config

```diff
@@ -30,6 +30,13 @@ ECHO_DEPS="                                                                 \
         $ngx_addon_dir/src/ngx_http_echo_foreach.h                          \
         "
 
+# nginx won't have HTTP_POSTPONE_FILTER_MODULE & HTTP_POSTPONE_FILTER_SRCS
+# defined since 1.9.11
+if test -z "$HTTP_POSTPONE_FILTER_MODULE"; then
+    HTTP_POSTPONE_FILTER_MODULE=ngx_http_postpone_filter_module
+    HTTP_POSTPONE_FILTER_SRCS=src/http/ngx_http_postpone_filter_module.c
+fi
```

Comment:
Do you think it's better to replace `fi` here with `else`? I think it's better to skip the following code snippet altogether for nginx 1.9.11+. What do you think?